MinecraftClient/ChatBot.cs

```diff
@@ -990,6 +990,20 @@ protected bool MoveToLocation(Mapping.Location location, bool allowUnsafe = fals
             return Handler.MoveTo(location, allowUnsafe, allowDirectTeleport);
         }
 
+        protected bool ApproachTo(Mapping.Location location, int radius, int maxNumberOfLocationsToEvaluate=5, int minDistance=0, bool allowUnsafe = false) 
+        {
+            return Handler.ApproachTo(location, radius, maxNumberOfLocationsToEvaluate, minDistance, allowUnsafe);
```

Comment:
`maxNumberOfLocationsToEvaluate`: This parameter seems absent in `Handler.ApproachTo()` below? Instead, there is a `pathfindingThreads` parameter which is also an `int`.